File: app/src/main/java/com/ibroadlink/screen/IRemoteCallback.java
package com.ibroadlink.screen;


import com.ibroadlink.library.aidlink.annotation.RemoteInterface;

/**
 * Created by codezjx on 2018/3/13.<br/>
 */
@RemoteInterface
public interface IRemoteCallback {

    void onStart();

    void onValueChange(int value);
}

File: speech/src/main/java/com/ibroadlink/aidlink/speech/service/IRemoteCallback.kt
package com.ibroadlink.aidlink.speech.service

import com.ibroadlink.library.aidlink.annotation.In
import com.ibroadlink.library.aidlink.annotation.RemoteInterface

@RemoteInterface
interface IRemoteCallback {

    fun onUpdateVocabs(@In vocabsMap: Map<String, List<String>>)

    fun onReplyResult(nativeApi: String, result: String)

    fun onServiceBind(packageName: String)

    fun onServiceUnBind(packageName: String)
}